packages/flutter_tools/lib/src/base/net.dart

```diff
@@ -12,31 +13,46 @@ const int kNetworkProblemExitCode = 50;
 
 /// Download a file from the given URL and return the bytes.
 Future<List<int>> fetchUrl(Uri url) async {
-  printTrace('Downloading $url.');
+  int attempts = 0;
+  int duration = 1;
+  while (true) {
+    attempts += 1;
+    final List<int> result = await _attempt(url);
+    if (result != null)
+      return result;
+    printStatus('Download failed -- attempting retry $attempts in $duration second${ duration == 1 ? "" : "s"}...');
```

Comment:
or you could just use "s" to represent seconds both singular and plural 😛

Reply:
that's crazy talk!